mkdocs.yml

```diff
@@ -1,4 +1,4 @@
-site_name: HIS4936 Final Project Starter
+site_name: Easter Rising Project
 theme_dir: theme
 theme: null
 copyright: 'Copyright &copy; 2018 <a href="mailto:dave.a.base@gmail.com">David J. Thomas</a>'
```

Comment:
You have an error that prevents a successful built in your mkdocs.yml. Where you list pages you have "- Workflow: 'word_flow.md'" but it should say 'work_flow.md'. Fix this if you want it to work on readthedocs.io